src/sentry/runner/commands/createuser.py

```diff
@@ -38,9 +38,12 @@ def _get_superuser():
 
 
 def _set_user_permissions(user):
-    from sentry.models import UserPermission
+    from sentry.models import UserPermission, UserRole, UserRoleUser
 
-    UserPermission.objects.create(user=user, permission="users.admin")
+    if click.confirm("Should this user have Super Admin permissions?", default=False):
+        role = UserRole.objects.get(name="Super Admin")
+        UserRoleUser.objects.create(user=user, role=role)
+        UserPermission.objects.create(user=user, permission="Super Admin")
```

Comment:
I believe you don't need to create this UserPermissions object, as the User will receive permissions through the `UserRole`

Reply:
yeah thats correct - the role is combined w/ the permission table to give a full set of permissions when they are looked up

Reply:
yup it checks if the user has the role first, ill get ride of creating UserPermission Obj